### DISCUSSION TITLE
关于联机的建议😊

### DISCUSSION THREAD

**Discussion by Starry-Sky-World** [Meloong-Git/PCL#948] [提问] [answered]
@LTCatt ，我觉得可以通过forge来去建设一个属于自己的服务器

**Comment by WorldHim** [comment, marked as answer]
> @LTCatt ，我觉得可以通过forge来去建设一个属于自己的服务器

首先，应该交到issue里面。
其次，这个和联机没什么关系吧，联机不等于开服。
还有，不要啥事没有乱@龙猫。
谢谢配合。

**Reply by Starry-Sky-World** [reply]
o

**Comment by miner233** [comment]
这种方法目前不可能。
PCL2/BakaXL/HMCL 已经联合宣布一起开发可以跨启动器联机的联机方案，而不是开服务器。

**Comment by IF-desu** [comment]
开服务器所占用的资源远比局域网联机多，而且为什么要用forge这个阴间玩意？
不可能做的
